### PR TITLE
IW-3397 | h1 sections support

### PR DESCRIPTION
**Pull request by latata**
https://wikia-inc.atlassian.net/browse/IW-3397
h1 section support + edit sections styling changes (but links still hidden)

### CODE REVIEW

File: app/components/article-content.js [hunk 1]
```diff
@@ -530,15 +530,14 @@ export default Component.extend(
     handleCollapsibleSectionHeaderClick(event) {
       const header = event.currentTarget;
 
+      if (event.target.closest('.section-edit-link')) {
```
Comment (xkxd) [inline app/components/article-content.js:533]: why do we need this here? maybe some comment would be helpful?

File: app/styles/module/article/_section-header.scss [hunk 2]
```diff
@@ -7,6 +8,42 @@ h2[section] {
...
+    display: flex;
+    height: 44px;
+    padding: 9px 13px;
+  }
+}
+
+.hide-edit-links {
+  h1[section],
+  h2[section] {
+    .section-edit-link,
```
Comment (xkxd) [inline app/styles/module/article/_section-header.scss:42]: could be
```
@at-root .hide-edit-links & {}
```
in the respective blocks above

**Review by xkxd — APPROVED** [review]

File: app/components/article-content.js (hunk 1 above)
Comment (rybmat) [reply app/components/article-content.js:533]: to not collapse / uncollapse section if you click on edit link

File: app/styles/module/article/_section-header.scss (hunk 2 above)
Comment (rybmat) [reply app/styles/module/article/_section-header.scss:42]: I'm not scss wizzard but @at-root makes code less readable to me

**Review by rybmat — APPROVED** [review]

**Pull request merged**